pkg/rehearse/jobs.go

```diff
@@ -1015,3 +1024,15 @@ func (e *Executor) submitPresubmit(job *prowconfig.Presubmit) (*pjapi.ProwJob, e
 	e.loggers.Job.WithFields(pjutil.ProwJobFields(&prowJob)).Info("Submitting a new prowjob.")
 	return &prowJob, e.pjclient.Create(context.Background(), &prowJob)
 }
+
+func determineJobURLPrefix(plank prowconfig.Plank, org string, repo string) string {
+	jobURLPrefix := plank.JobURLPrefixConfig["*"]
```

Comment:
Do we need a nil check on `JobURLPrefixConfig`?

Reply:
Technically no, as it's defined now, and removing it would break a lot of functionality. I toyed with the idea of adding one here, but if it were to be removed in the future (due to deprecation) we would want to blow up here so we could restore functionality.

Reply:
Sounds good enough to me.